cdap-proto/src/main/java/co/cask/cdap/proto/PluginDetail.java

```diff
@@ -19,20 +19,20 @@
  * Represents a plugin id and name in an HTTP response.
  */
 public class PluginDetail {
-  private final String pluginId;
-  private final String pluginName;
+  private final String id;
+  private final String name;
 
-  public PluginDetail(String pluginId, String pluginName) {
-    this.pluginId = pluginId;
-    this.pluginName = pluginName;
+  public PluginDetail(String id, String name) {
```

Comment:
can we include type too? Otherwise its hard to tell if its Table batchsource or Table batchsink for example.